### DISCUSSION TITLE
add fbprophet package in environment

### DISCUSSION THREAD

**Discussion by Chaodong** [PaddlePaddle/PaddleSpatial#174] [BaiduKDDCup2022] [answered]
is it possible to add fbprophet package in environment?
The framework is paddlepaddle, but fbprophet is used for component forecast

**Comment by ramber1836** [comment, marked as answer]
The prophet package is incompatible with the current paddlepaddle environment. Please refer to https://github.com/PaddlePaddle/PaddleSpatial/discussions/165 .
